modules/graphdb/templates/01_disk_management.sh.tpl

```diff
@@ -1,184 +1,123 @@
-#!/usr/bin/env bash
+#!/bin/bash
 
-# This script performs the following actions:
-# * Set common variables: Retrieves IMDS token, instance ID, and availability zone.
-# * Search for available EBS volume: Iterates up to 6 times to find or create an EBS volume tagged for GraphDB.
-# * Attach EBS volume to instance: Connects the volume to the EC2 instance using a specified device name.
-# * Store volume ID: Saves the volume ID for use in another script.
-# * Handle EBS volume for GraphDB data directory: Manages device mapping complexities and creates a file system if needed.
-# * Mount and configure file system: Ensures the file system is mounted, configures automatic mounting, and sets proper ownership.
```

Comment:
Can't get why you have changed the shebang, removed the description and updated the set command to a less expressive form?

Reply:
my mistake, will fix

Reply:
fixed